Runtimes/Core/CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ add_compile_definitions(
   $<$<BOOL:${SwiftCore_ENABLE_UNICODE_DATA}>:-DSWIFT_STDLIB_ENABLE_UNICODE_DATA> # Stubs
   $<$<BOOL:${SwiftCore_ENABLE_ENVIRONMENT}>:-DSWIFT_STDLIB_HAS_ENVIRON> # Concurrency, runtime, shims, platform overlay
   $<$<BOOL:${SwiftCore_ENABLE_BACKDEPLOYMENT_SUPPORT}>:-DSWIFT_STDLIB_SUPPORT_BACK_DEPLOYMENT> # Concurrency, Compatibility override, magic symbols
+  $<$<BOOL:${SwiftCore_ENABLE_STDLIB_TRACING}>:-DSWIFT_STDLIB_TRACING>
+  $<$<BOOL:${SwiftCore_ENABLE_STDLIB_TRACING}>:-DSWIFT_STDLIB_CONCURRENCY_TRACING>
```

Comment:
This is really getting crazy. Any plans on splitting this up @etcwilde @justice-adams-apple?

Reply:
If it only affects the individual libraries, they should probably be attached to that specific library.
I don't mind as a quick fix, but in that case, can we include a comment specifying which libraries it is affecting?
Thanks.

Reply:
@etcwilde moved the compile definitions into the runtime cmake target